### PR TITLE
Fix #1168 - Add a modified version of CONTRIBUTING.md

### LINKED ISSUE #1168: change CONTRIBUTING.md
The current file is out of date for an Apache podling.

I'm not sure what an updated file would say but it should probably mention that CLAs may be required for big changes.

https://www.apache.org/licenses/contributor-agreements.html

### PR DESCRIPTION
**Pull request by rzo1**
as the title says

### CODE REVIEW

**Review by jnioche — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,66 +1,59 @@
-Contributing to StormCrawler
+Contributing to Apache StormCrawler
 ============================
 
+Thank you for your intention to contribute to Apache StormCrawler. As an open-source community, we highly appreciate external contributions to our project.
```
Comment (ayushtkn) [inline CONTRIBUTING.md:4]: do we need 'external' here, who is considered internal then?

**Review by ayushtkn — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (rzo1) [reply CONTRIBUTING.md:4]: Let's drop external :-)

Comment (rzo1) [reply CONTRIBUTING.md]: Fixed @ayushtkn - thx for review!

**Pull request merged**